dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -237,6 +240,29 @@ private static void checkTokens(final Token quayToken, final Token githubToken,
         }
     }
 
+
+    @PUT
+    @Path("/{containerId}/insertDescriptorType")
+    @Timed
+    @UnitOfWork
```

Comment:
Feels weird, but maybe it makes sense to immediately put a deprecated annotation on this and . 
If I understand this correctly, this endpoint would only get used by us after the release?
And put a `@RolesAllowed({ "curator", "admin" })` on it